app/src/main-process/menu/build-default-menu.ts

```diff
@@ -159,6 +159,13 @@ export function buildDefaultMenu({
         accelerator: 'CmdOrCtrl+A',
         click: emit('select-all'),
       },
+      separator,
+      {
+        id: 'find',
+        label: __DARWIN__ ? 'Find' : '&Find',
```

Comment:
should this say 'Find in Diff'? or something like that to indicate the scope of the find?

Reply:
That's a great question. I went back and forth a bit before I picked the `Find` label. The primary thing that tipped the balance for `Find` vs `Find in Diff` was platform familiarity/consistency. Most apps that I looked at on Mac and Windows used Find unless there were multiple separate things one could search and even when there were multiple things to search, such as in VS Code, the primary action was still labelled `Find` and the secondary `Find in Files` with a separate shortcut.

So I'm moderately in favor of keeping it as just `Find` for now but I obviously welcome more input here.

Reply:
thanks for the context @niik. sounds good to me!